server/aligners.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/python3
+
+import itertools
+import logging
+from os import path
+import os
```

Comment:
This two lines consecutives are a bit strange. If you need all os module (I don't think so) you should use path as os.path

Reply:
I added `from os import path` because I use `path` quite a bit but still use other members from `os`, so I imported `path` directly just for convenience.